src/pages/Profile.jsx

```diff
@@ -1,26 +1,32 @@
 import { useAuth } from "@/models";
+import { Outlet } from "react-router-dom";
 
 export default function Profile() {
   const auth = useAuth();
+  const parseName = (name) => {
+    let nameArray = name.split(" ");
+    if (nameArray.length > 1) {
+      return nameArray[0][0] + " " + nameArray[1][0];
+    }
+    return nameArray[0];
+  };
```

Comment:
if this is a constant function, move it out of the react function. performance improving